deploy-postgres-exporter.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+
+# Description: Export Prometheus metrics from associated Postgres instance
+#
+# Disk: none
+# Network: 100mbps
+# Liveness probe: n/a
+# Ports exposed to other Sourcegraph services (Prometheus target): 9187/TCP
+# Ports exposed to the public internet: none
+#
+docker run --detach \
+    --name=postgres-exporter \
+    --network=sourcegraph \
+    --restart=always \
+    --cpus=1 \
+    --memory=1g \
+    -e DATA_SOURCE_NAME="postgresql://sourcegraph:sourcegraphd@pgsql:5432/postgres?sslmode=disable" \
```

Comment:
@uwedeportivo Can a user specify PG* variables here individually instead of a DSN? Composing the DSN requires hard-coding the password or some weird trickery if it contains values that need to be URL-encoded

I imagine the same would be true in deploy-sourcegraph; we should probably just use the PG* variables mirroring the frontend